<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddExhibitDialog</class>
 <widget class="QDialog" name="AddExhibitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add exhibit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Exhibit train images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectTrainImagesPushButton">
       <property name="text">
        <string>Select images</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="trainImagesPathsLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Exhibit main image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mainImageSelectPushButton">
       <property name="text">
        <string>Select image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mainImagePathLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Exhibit title</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exhibitTitleLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Exhibit description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="exhibitDescriptionTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
